.eslintrc

```diff
@@ -22,16 +22,16 @@
         "no-fallthrough": "off",
         "prettier/prettier": "warn",
         "indent": [
-            "error",
+            "warn",
```

Comment:
Why are we changing error levels?

Reply:
I just believed small **stylistic** linting rules aren't worth something like failing a CI test, or stops a build entirely if linting were checked.

We could also simply revert this and leave rules more strict for the future.

Reply:
I wouldn't change the levels. Failing a CI test and stopping a build is a way to enforce the rules and to be 100% sure that the code is of high quality.

If we think that some rules are less important than the other ones, I would just remove them.